engine/core/IgeTexture.ts

```diff
@@ -74,9 +72,9 @@ export class IgeTexture extends IgeAsset {
 			ige.textures.add(id, this);
 		}
 
-		this.dependencies.add("IgeImageClass", import("./IgeImage.js").then(({ IgeImage: IgeModule }) => {
-			IgeImageClass = IgeModule;
-		}));
+		// this.dependencies.add("IgeImageClass", import("./IgeImage.js").then(({ IgeImage: IgeModule }) => {
+		// 	IgeImageClass = IgeModule;
+		// }));
```

Comment:
If IgeImage is no longer required in this class, this code can safely be removed. It was added because of a circular reference between IgeTexture and IgeImage.